### PR TITLE
Fix GeneralName

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

**Review by henkbirkholz — APPROVED** [review]
thx! lgtm

File: cddl/tst-info.cddl [hunk 1]
```diff
@@ -21,11 +21,11 @@ MessageImprint = [
...
+  1 => ~time
   ? -8 => profiled-duration
   * int => any
 }
 profiled-duration = {* int => any}
 
-GeneralName = ( GeneralNameType : int, GeneralNameValue : any )
-SingleEntryGeneralNames = [ 1*1 GeneralName ]
+GeneralNameAttr = ( GeneralNameType : int, GeneralNameValue : any )
+GeneralName = [ + GeneralNameAttr ]
```
Comment (thomas-fossati) [inline cddl/tst-info.cddl:31]: hmm, this is not what 3161 says though: `tsa` is `GeneralName` singular, not `GeneralNames` plural...

Comment (cabo) [reply cddl/tst-info.cddl:31]: Yes.  that is what the PR says?

Comment (cabo) [reply cddl/tst-info.cddl:31]: Now I see what you were trying to do... RFC 5280...

Comment (cabo) [reply cddl/tst-info.cddl:31]: Fixed.

Comment (cabo) [reply cddl/tst-info.cddl:31]: https://github.com/ietf-rats/draft-birkholz-rats-epoch-marker/pull/20

Comment (thomas-fossati) [reply cddl/tst-info.cddl:31]: thanks!

**Pull request merged**